Block/Checkout/Onepage/Success/AdditionalInformation.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Omise\Payment\Block\Checkout\Onepage\Success;
+
+class AdditionalInformation extends \Magento\Framework\View\Element\Template
+{
+    /**
+     * @var \Magento\Checkout\Model\Session
+     */
+    protected $_checkoutSession;
+
+    /**
+     * @var string
+     */
+    protected $paymentType;
+
+    /**
+     * @var string
+     */
+    protected $paymentData;
+
+    /**
+     * @param \Magento\Framework\View\Element\Template\Context $context
+     * @param \Magento\Checkout\Model\Session $checkoutSession
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Framework\View\Element\Template\Context $context,
+        \Magento\Checkout\Model\Session $checkoutSession,
+        array $data = []
+    ) {
+        $this->_checkoutSession = $checkoutSession;
+        $this->order            = $this->getOrder();
+        $this->paymentData      = $this->order->getPayment()->getData();
+        $this->paymentType      = $this->getPaymentAdditionalInformation('payment_type');
+        parent::__construct($context, $data);
+    }
+
+    /**
+     * @return \Magento\Sales\Model\Order
+     */
+    protected function getOrder() {
```

Comment:
Just minor code style (normally we break 1 line for `{`
(and same to all the below lines)